test/companies/jobs_test.exs

```diff
@@ -3,7 +3,7 @@ defmodule Companies.JobsTest do
 
   import ExUnit.CaptureLog
 
-  alias Companies.Jobs
+  alias Companies.{Jobs}
```

Comment:
We don't need `{}` here since we're only aliasing a single module.

Reply:
Fixed